types/sanctuary/index.d.ts

```diff
@@ -29,16 +30,28 @@ type Predicate<A> = (a: A) => boolean;
 
 interface StrMap<A> { [k: string]: A; }
 
+interface Pair<A, B> {
+  constructor: {
+    '@@type': 'sanctuary-pair/Pair@1';
+  };
+  'fantasy-land/equals'(other: Pair<A, B>): boolean;
+  'fantasy-land/lte'(other: Pair<A, B>): boolean;
```

Comment:
Wouldn't it be easier than repeating these methods all over:
```
interface Pair<A, B> extends IOrd<Pair<A, B>> {
  ...
}
```
?

Reply:
Good idea. Updated. :)